src/agent/src/device/mod.rs

```diff
@@ -1192,9 +1192,9 @@ mod tests {
         );
         spec.set_annotations(Some(annotations));
 
-        // create a file in /tmp/cdi with nvidia.json content
-        let cdi_dir = PathBuf::from("/tmp/cdi");
-        let cdi_file = cdi_dir.join("kata.json");
+        // Create a temporary directory for CDI￼
```

Comment:
There's some weird character at the end of this line...

Reply:
Fixed.